### PR TITLE
Fix the "Show full source" positioning

### PR DESCRIPTION
**Pull request by cwayfinder**

### CODE REVIEW

File: code-example.html [hunk 1]
```diff
@@ -48,7 +48,8 @@
   }
   .source-toggle {
     cursor: pointer;
-    margin-right: -40px;
+    position: absolute;
+    right: 0;
```
Comment (jouni) [inline code-example.html:53]: This will only work if the parent is `position: relative;`.

``` css
.code {
  ...
  position: relative;
}
```

**Comment by jouni** [comment]
The `.copy` button styles can also be made so that they don’t require magic numbers (using transform):

``` css
.copy.code-example {
  border-left: 1px solid rgba(0, 0, 0, 0.1);
  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
  border-radius: 0 0.3em 0 0.3em;
  padding: 3px;
  display: inline-block;
  position: relative;
  z-index: 1;
  top: 1px;
  right: 1px;
  transform: translatey(100%);
  -webkit-transform: translatey(100%);
}
```

**Comment by cwayfinder** [comment]
With `transform: translatey(100%);` I get into such trouble http://prntscr.com/79xyi9

**Comment by cwayfinder** [comment]
Now it works

File: code-example.html [hunk 2]
```diff
@@ -28,10 +28,11 @@
     padding-left: 2em;
     width: 100%;
     overflow: hidden;
+    position: relative;
   }
 
   .source {
-    display: none;
+    display: none !important;
```
Comment (jouni) [inline code-example.html:35]: Whoops, this was not intended I suppose. Now the "show full source" is never visible. I’ll fix it.

**Pull request merged**